__tests__/services/TransactionFilterService.spec.ts

```diff
@@ -19,7 +19,7 @@ import { getTestAccount } from '@MOCKS/Accounts';
 import { TransferTransaction } from 'symbol-sdk';
 
 const currentSigner = getTestAccount('remoteTestnet');
-const recepient = getTestAccount('remoteMijin');
+const recepient = getTestAccount('remoteTestnet2');
```

Comment:
You may want to adjust the names to be more descriptive here, for readability. How about remoteTestnetReceiver and remoteTestnetSender?